test/wdio_functional/page_objects/console/console_page.js

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import expect from 'expect';
+import {
+  createTestSubjectSelectorCss,
+  createTestSubjectSelectorXpath
+} from '../../helpers/helpers';
+import BasePage from '../common/base_page';
+
+export default class ConsolePage extends BasePage {
+
+  constructor(driver) {
+    super(driver);
+    this.breadCrumbSelector = 'div.kuiLocalBreadcrumb';
+    this.requestEditorSelector = createTestSubjectSelectorCss('request-editor');
+    this.editorContentSelector = 'div.ace_scroller > div.ace_content';
+    this.requestEditorTextSelector =
+      `${this.requestEditorSelector} > ${this.editorContentSelector}`;
+    this.playButtonSelector = createTestSubjectSelectorCss('send-request-button');
+    this.responseEditorSelector = '#output';
+    this.responseEditorTextSelector =
+      `${this.responseEditorSelector} > ${this.editorContentSelector}`;
+    this.consoleSettingsButtonSelector =
+      `//button${createTestSubjectSelectorXpath('consoleSettingsButton')}`;
```

Comment:
I don't understand the switching between css and xpath selectors.  I think we've avoided using xpath selectors in the ftr.

Reply:
I was having trouble actually accessing the element with CSS even when using the `data-test-subj`. Why did you avoid XPATH in `functional_test_runner`?

Reply:
I think it's fair to say that just about everyone on the team understands how CSS selectors work and the opposite is true about XPath